src/goto-instrument/dump_c.cpp

```diff
@@ -935,6 +1025,9 @@ void dump_ct::convert_global_variable(
       !converted_global.insert(symbol.name).second)
     return;
 
+  if(func.empty() || symbol.is_extern)
```

Comment:
I don't really understand this condition. For example, is this function supposed to just convert global symbols and therefore we must first check either the containing function `func` is empty OR the symbol is external. Is there a reason why this condition  does not include the possibility of `symbol.value.is_not_nil()`.